cmp/grid/GridModel.js

```diff
@@ -315,6 +317,13 @@ export class GridModel {
         this.compact = compact;
     }
 
+    /** Return the minimum row height specified by the columns (if any) */
+    @computed
```

Comment:
What's the value of exposing this on GridModel?  Can this very same getter be an implementation bit on the grid?   Can then easily combine it with the other logic around row height that is there.

 We don't give the creator of a GridModel any way to specify this at a gridmodel level in any case